src/commands/rps.ts

```diff
@@ -0,0 +1,193 @@
+import { CommandInteraction, TextBasedChannel, TextChannel, User } from 'discord.js';
+import {
+  type ArgsOf,
+  Discord,
+  On,
+  SimpleCommand,
+  SimpleCommandMessage,
+  SimpleCommandOption,
+  Slash,
+  SlashOption,
+} from 'discordx';
+
+@Discord()
+class RPS {
+  private general_channel = '103678524375699456';
+
+  private timeout_duration = 1000 * 5;
```

Comment:
This should be 5 minutes not 5 seconds
```suggestion
  private timeout_duration = 1000 * 60 * 5
```

Reply:
my bad lol